src/_posts/databases/postgresql/2000-01-01-troubleshooting.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Troubleshooting Scalingo for PostgreSQL
+nav: Troubleshooting
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 9
+---
+
+## Identifying Performances Issues
+
+
+## Understanding Idle Queries
```

Comment:
It seems that you have inverted the titles of the 2 paragraphs. This paragraph seems to discuss memory consumption and not idle connections